storage/badger/cache.go

```diff
@@ -94,12 +94,16 @@ func (c *Cache) Get(key interface{}) func(*badger.Txn) (interface{}, error) {
 		}
 
 		// get it from the database
-		c.metrics.CacheMiss(c.resource)
 		resource, err := c.retrieve(key)(tx)
+		if errors.Is(err, storage.ErrNotFound) {
```

Comment:
Its my being picky - we can do this check only after `err` is not `nil`, therefore saving one or two CPU cycles - I think this happen often enough to be worth it

Reply:
makes sense. updated